packages/loader/container-loader/src/container.ts

```diff
@@ -129,6 +135,11 @@ export interface IContainerLoadOptions {
      * Loads the Container in paused state if true, unpaused otherwise.
      */
     loadMode?: IContainerLoadMode;
+    /**
+     * Create the container on load, without an existing snapshot.
+     * Used only for supporting legacy scenarios.
+     */
+    createOnLoad?: boolean;
```

Comment:
Same here - we should be clear that this is only temp escape valve, all the places that are changes to enable this workflow needs to be marked as deprecated to set right expectations